frontend/craco.config.js

```diff
@@ -0,0 +1,14 @@
+const path = require('path');
```

Comment:
Why should we use `craco` here?

Reply:
Because I use Create-react-app for initial setup. If I not use craco, my allias import path will not work. 
 [I read and fixed it following this blog](https://plusreturn.com/blog/how-to-configure-a-path-alias-in-a-react-typescript-app-for-cleaner-imports/)